src/Airy/cairy.jl

```diff
@@ -372,20 +410,57 @@ end
     return A, B, C, D
 end
 
+@generated function airyaix_levin(x::Complex{T}, ::Val{N}) where {T <: Union{Float32, Float64}, N}
+    :(
+        begin
+            xsqr = sqrt(x)
+            out = zero(typeof(x))
+            t = GAMMA_ONE_SIXTH(T) * GAMMA_FIVE_SIXTHS(T) / 4
+            a = inv(4*xsqr*x)
```

Comment:
`4/(xsqr*x)`

Reply:
Could be mistaken but are those equivalent ? Would you prefer `a = 1 / (4*xsqr*x)`? Or `0.25 / (xsqr*x)`?

Reply:
oh, duh. `.25/...` (it's very minor but saves a multiplication)